1038-vertical-nav-design-guidance.md

```diff
@@ -0,0 +1,36 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-18
+updated: 2024-10-21
+---
+
+# Vertical Nav Component
+
+The [vertical nav](https://clarity.design/documentation/vertical-nav) component provides vertically-aligned navigation links. It can fit as many navigation links as needed by scrolling when the content exceeds the viewport.
+
+
+## Guidance
+
+- **Must** use to collapse and expand the navigation to free up space for the content area.
+- **Must** use dividers and groups to separate logical clusters of navigation items.
+- **Must not** use vertical navigation with a header and a sub navigation menu. Having too many places for navigation will be confusing for users.
+
+## Nav Icons
+
+- **Must** maintain consistency across using icons on navigation items. Avoid using icons to some of top navigation items and not to others.
+- **Must** apply icons to top-level navigation items. *Must not** apply icons to child links.  
+- **Must** place icons to the left of the top navigation labels.
+- **Should** use icons to describe navigation items which are helpful when the navigation is collapsed.
+
```

Comment:
```suggestion
```

extra blank line here. This breaks the list rendering: https://27--clarity-guidance.netlify.app/1038